packages/agreement-process/src/services/readmodel/readModelService.ts

```diff
@@ -116,9 +136,9 @@ const getAgreementsFilters = (
       }),
     ...(attributeId && {
       $or: [
-        { "data.certifiedAttributes": { $elemMatch: { id: attributeId } } },
-        { "data.declaredAttributes": { $elemMatch: { id: attributeId } } },
-        { "data.verifiedAttributes": { $elemMatch: { id: attributeId } } },
+        makeAttributesFilter("certifiedAttributes", attributeId),
```

Comment:
Fixing a bug spotted writing the tests: `attributeId` filter was not working properly